example-apps/html-js/shared/styles.css

```
@@ -0,0 +1,242 @@
+/* Shared styles for all Img2Num example apps.
+   Tokens mirror the main site's custom.css (img2num.dev) so the demos feel
+   like part of the site, without depending on the Docusaurus build's
+   hash-named stylesheet. If the site palette changes, update these values. */
+
+:root {
+  --font-sans: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Tahoma" to be "tahoma" (value-keyword-case)

(value-keyword-case)